### PR TITLE
fix: kyma2 gardener minor versions upgrade

### PR DESCRIPTION
**Pull request by adityabhatia**
**Description**

Changes proposed in this pull request:

- Fix unsupported arithmetic operation
- Include log for version overview

**Related issue(s)**

### CODE REVIEW

**Review by jakobmoellersap — APPROVED** [review]

File: prow/scripts/lib/kyma.sh [hunk 1]
```diff
@@ -140,12 +140,11 @@ function kyma::get_offset_minor_releases() {
     # PREVIOUS_MINOR_VERSION_COUNT - Count of last Kyma2 minor versions to be upgraded from
     for i in $(seq 1 "$PREVIOUS_MINOR_VERSION_COUNT"); do
         if [ "$MINOR" -gt 0 ]; then
-            ((MINOR-=1))
+            MINOR=$((MINOR-1))
```
Comment (PK85) [inline prow/scripts/lib/kyma.sh:143]: this is not enough. Solution could be: decrease minor and search for newest patch and then use that version

**Pull request merged**